### PR TITLE
Tests for PaymentInstruments.

### PR DESCRIPTION
**Pull request by rsolomakhin**
@marcoscaceres: Please take a look at this initial set of tests for the `PaymentInstruments` interface. Thank you!

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/24149)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/339627031)

File: payment-handler/payment-instruments.https.html [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+            src: '/images/rgrg-256x256.png',
+            sizes: '256x256',
+            type: 'image/pn' + 'g'.repeat(1000),
+          },
+        ],
+        enabledMethods: ['basic-card'],
+      },
+    );
+    return promise_rejects(t, new TypeError(), setPromise);
+  }, 'Cannot register instruments with very long icon media type image/pngggggg...');
```
Comment (marcoscaceres) [inline payment-handler/payment-instruments.https.html]: This restriction seems odd? is this in the MIME spec?

File: payment-handler/payment-request-event.https.html [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+<meta charset="utf-8">
+<title>Tests for PaymentRequestEvent</title>
+<link rel="help" href="https://w3c.github.io/payment-handler/#the-paymentrequestevent">
+<link rel="manifest" href="/payment-handler/basic-card.json">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="register-and-activate-service-worker.js"></script>
+<p>When the payment sheet is shown, please authorize the mock payment.</p>
+<script>
+// Use a different scope for the payment handler, because Chrome closes all
```
Comment (marcoscaceres) [inline payment-handler/payment-request-event.https.html]: This comment seems Chrome specific?

**Review by marcoscaceres — APPROVED** [review]
LGTM.

**Comment by marcoscaceres** [comment]
Couple of questions, but overall the tests seem good.

File: payment-handler/payment-instruments.https.html (hunk 1 above)
Comment (rsolomakhin) [reply payment-handler/payment-instruments.https.html]: My bad. the description really should be "Don't crash when registering instrument with very long icon media type image/pngggg..."

File: payment-handler/payment-request-event.https.html (hunk 2 above)
Comment (rsolomakhin) [reply payment-handler/payment-request-event.https.html]: Removed

**Comment by rsolomakhin** [comment]
@marcoscaceres: Thank you for the review! I've addressed your comments. Please take another look.

**Review by marcoscaceres — APPROVED** [review]

**Pull request merged**